cape-char.el

```diff
@@ -25,44 +25,35 @@
 ;;; Code:
 
 (require 'cape)
+(require 'quail)
```

Comment:
We shouldn't require `quail` here, such that we don't load it at runtime. Instead the require should go to the `cape-char--translation` function.

Reply:
[8a58051](https://github.com/minad/cape/pull/91/commits/8a5805143e5b84855b7be9b196224e657c8b90b9)